portfolio/src/main/webapp/WEB-INF/meme-trend.csv

```diff
@@ -0,0 +1,199 @@
+2004-01,1
+2004-02,1
+2004-03,1
+2004-04,1
+2004-05,1
+2004-06,1
+2004-07,1
+2004-08,1
+2004-09,2
+2004-10,2
+2004-11,1
+2004-12,2
+2005-01,2
+2005-02,2
+2005-03,1
+2005-04,2
+2005-05,2
+2005-06,2
+2005-07,1
+2005-08,2
+2005-09,2
+2005-10,2
+2005-11,2
+2005-12,2
+2006-01,2
+2006-02,2
+2006-03,2
+2006-04,2
+2006-05,2
+2006-06,2
+2006-07,2
+2006-08,2
+2006-09,2
+2006-10,2
+2006-11,2
+2006-12,2
+2007-01,2
+2007-02,2
+2007-03,2
+2007-04,2
+2007-05,3
+2007-06,3
+2007-07,2
+2007-08,2
+2007-09,3
+2007-10,3
+2007-11,3
+2007-12,3
+2008-01,3
+2008-02,3
+2008-03,3
+2008-04,3
+2008-05,3
+2008-06,3
+2008-07,2
+2008-08,3
+2008-09,3
+2008-10,3
+2008-11,3
+2008-12,3
+2009-01,3
+2009-02,3
+2009-03,4
+2009-04,3
+2009-05,4
+2009-06,4
+2009-07,3
+2009-08,3
+2009-09,4
+2009-10,4
+2009-11,4
+2009-12,4
+2010-01,4
+2010-02,4
+2010-03,4
+2010-04,4
+2010-05,4
+2010-06,4
+2010-07,4
+2010-08,5
+2010-09,5
+2010-10,5
+2010-11,6
+2010-12,6
+2011-01,6
+2011-02,7
+2011-03,8
+2011-04,8
+2011-05,8
+2011-06,8
+2011-07,8
+2011-08,9
+2011-09,10
+2011-10,11
+2011-11,14
+2011-12,17
+2012-01,19
+2012-02,25
+2012-03,25
+2012-04,27
+2012-05,25
+2012-06,25
+2012-07,24
+2012-08,26
+2012-09,24
+2012-10,24
+2012-11,24
+2012-12,25
+2013-01,24
+2013-02,26
+2013-03,26
+2013-04,27
+2013-05,26
+2013-06,25
+2013-07,24
+2013-08,27
+2013-09,26
+2013-10,26
+2013-11,26
+2013-12,29
+2014-01,30
+2014-02,31
+2014-03,31
+2014-04,32
+2014-05,33
+2014-06,35
+2014-07,37
+2014-08,36
+2014-09,35
+2014-10,39
+2014-11,45
+2014-12,53
+2015-01,44
+2015-02,48
+2015-03,50
+2015-04,48
+2015-05,48
+2015-06,51
+2015-07,49
+2015-08,53
+2015-09,62
+2015-10,56
+2015-11,59
+2015-12,62
+2016-01,58
+2016-02,61
+2016-03,57
+2016-04,61
+2016-05,65
+2016-06,69
+2016-07,66
+2016-08,68
+2016-09,78
+2016-10,74
+2016-11,78
+2016-12,77
+2017-01,77
+2017-02,80
+2017-03,79
+2017-04,80
+2017-05,80
+2017-06,78
+2017-07,74
+2017-08,75
+2017-09,77
+2017-10,72
+2017-11,71
+2017-12,72
+2018-01,74
+2018-02,73
+2018-03,75
+2018-04,75
+2018-05,78
+2018-06,77
+2018-07,80
+2018-08,75
+2018-09,80
+2018-10,81
+2018-11,83
+2018-12,84
+2019-01,88
+2019-02,84
+2019-03,80
+2019-04,84
+2019-05,85
+2019-06,79
+2019-07,79
+2019-08,80
+2019-09,83
+2019-10,84
+2019-11,89
+2019-12,84
+2020-01,85
+2020-02,85
+2020-03,97
+2020-04,100
+2020-05,91
+2020-06,85
+2020-07,83
```

Comment:
nit: always include an endline on all of your files, even data files.

Reply:
Done!